test/__snapshots__/index.spec.js.snap

```diff
@@ -49,20 +49,7 @@ Array [
 ]
 `;
 
-exports[`Luxafor exec Should execute the write for only one led position 1`] = `
-Array [
-  Array [
-    1,
-    255,
-    0,
-    255,
-    0,
-    0,
-    0,
-    0,
-  ],
-]
-`;
+exports[`Luxafor exec Should execute the write for only one led position 1`] = `Array []`;
```

Comment:
I don't think this is expected for this test. Should it have written to the entire lux instead?

```js
    it('Should execute the write for only one led position', () => {
      const instance = setup()
        .color(0, 255, 0)
        .exec();

      expect(
        instance.device.data
      ).toMatchSnapshot();
    });
```